src/Assignment.ts

```diff
@@ -0,0 +1,59 @@
+import Question from "./model/Question";
+import alasql from "alasql";
+
+const question = new Question(
+    // Prompt for the user
+    `Select all columns from the "Employees" table.`,
+
+    // SQL queries to configure the context
+    `CREATE TABLE Employees (ID INT, Name VARCHAR(255), Supervisor INT);
+    INSERT INTO Employees VALUES
+        (1, "Joe", NULL),
+        (2, "Bill", 1),
+        (3, "Jill", NULL),
+        (4, "Bob", 3),
+        (5, "Marvin", 1);`,
+
+    // Answer key query
+    `SELECT * FROM Employees`);
+
+function compareResults(a: Array<any>, b: Array<any>) {
+    if (a.length != b.length) return false;
+
+    // TODO: This isn't ideal, but it gets the job done for the time being. Write something better in the future.
+    return JSON.stringify(a) === JSON.stringify(b);
```

Comment:
This is brilliant, its so quick and simple!

Reply:
I'm glad that it works!